src/traffic_cache_tool/CacheTool.cc

```diff
@@ -1255,7 +1257,7 @@ Check_Freelist(std::string devicePath)
   if ((err = cache.loadSpan(SpanFile))) {
     cache.dumpSpans(Cache::SpanDumpDepth::SPAN);
     for (auto sp : cache._spans) {
-      if (devicePath.size() > 0 && 0 == strncmp(sp->_path.path(), devicePath.data(), devicePath.size())) {
+      if (devicePath.size() > 0 && 0 == strncmp(sp->_path.c_str(), devicePath.data(), devicePath.size())) {
```

Comment:
This is where the `ts::path::view` method would win - then it would be `&& sp->_path.view() == devicePath`.